### PR TITLE
[irods/irods#5239] Removed confusing failure message on a successful iinit (4-2-stable)

### PR DESCRIPTION
**Pull request by korydraughn**

### CODE REVIEW

File: src/iinit.cpp [hunk 1]
```diff
@@ -396,7 +390,6 @@ int main( int argc, char **argv )
             }
             else {
                 obj_to_dump = json_env;
-                std::cerr << "Failed to update [" << env_file << "]\n";
```
[the remark this answers is not in the document]
Comment (korydraughn) [reply src/iinit.cpp:399]: i added that when i removed the jansson json library. i believe i misinterpreted the original logic for this line.

[the remark this answers is not in the document]
Comment (korydraughn) [reply src/iinit.cpp:399]: this is the code before the jansson removal.

https://github.com/irods/irods_client_icommands/blob/a8c83caa7175f9fc32597a509b89d0c7cf10d516/src/iinit.cpp#L419-L438

[the remark this answers is not in the document]
Comment (trel) [reply src/iinit.cpp:399]: got it.

**Comment by trel** [comment]
if happy, please # it.

**Comment by korydraughn** [comment]
#'d

**Pull request merged**